src/wl_window.c

```diff
@@ -686,6 +686,12 @@ const char* _glfwPlatformGetKeyName(int key, int scancode)
     return NULL;
 }
 
+const int _glfwPlatformGetKeyScancode(int key)
+{
+    // TODO
```

Comment:
If you don't feel confident implementing this for Wayland and Mir, I'll make a commit for you to rebase on and we can add all platforms in one go.